manifests/m/MuhammedKalkan/OpenLens/6.0.1/MuhammedKalkan.OpenLens.installer.yaml

```
@@ -1,30 +1,32 @@
-# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.7-2-5
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: MuhammedKalkan.OpenLens
-PackageVersion: 6.0.1
-InstallerLocale: en-US
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-InstallModes:
-- interactive
-- silent
-UpgradeBehavior: install
-ReleaseDate: 2022-08-11
-AppsAndFeaturesEntries:
-  - DisplayVersion: 6.0.1-latest.1660157736010
-Installers:
-- Architecture: x64
-  Scope: user
-  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.0.1/OpenLens-6.0.1.exe
-  InstallerSha256: 20B2E7420BE69284D6CB35549A6507343DA52D0AB0AF4D72F0C74D7D6D63793B
-  InstallerSwitches:
-    Custom: /CURRENTUSER
-- Architecture: x64
-  Scope: machine
-  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.0.1/OpenLens-6.0.1.exe
-  InstallerSha256: 20B2E7420BE69284D6CB35549A6507343DA52D0AB0AF4D72F0C74D7D6D63793B
-  InstallerSwitches:
-    Custom: /ALLUSERS
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MuhammedKalkan.OpenLens
+PackageVersion: 6.0.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 6.0.1-latest.1660157736010

```

```suggestion
- DisplayVersion: 6.0.1-latest.1660667913150
```
This should match control panel.